x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/nvidia/embeddings/NvidiaEmbeddingsResponseHandler.java

```diff
@@ -26,4 +30,28 @@ public class NvidiaEmbeddingsResponseHandler extends OpenAiResponseHandler {
     public NvidiaEmbeddingsResponseHandler(String requestType, ResponseParser parseFunction) {
         super(requestType, parseFunction, ErrorResponse::fromResponse, false);
     }
+
+    @Override
+    protected void checkForFailureStatusCode(Request request, HttpResult result) throws RetryException {
+        super.checkForFailureStatusCode(request, result);
+    }
```

Comment:
Does this method need to be overridden? It just calls the super method without any additional logic.